open_discussions/settings.py

```diff
@@ -404,7 +404,7 @@
     'JWT_EXPIRATION_DELTA': datetime.timedelta(seconds=60*60),
     'JWT_ALLOW_REFRESH': True,
     'JWT_REFRESH_EXPIRATION_DELTA': datetime.timedelta(days=7),
-    'JWT_AUTH_COOKIE': 'open_discussions_jwt',
+    'JWT_AUTH_COOKIE': get_string('OPEN_DISCUSSIONS_COOKIE_NAME', 'open_discussions_jwt'),
```

Comment:
Should we make this and the micromasters environment variable required in `app.json`?

Reply:
Sure, I can do that.